src/storage/config_manager.rs

```diff
@@ -78,6 +78,7 @@ impl<EK: Engine, K: ConfigurableDb, L: LockManager> ConfigManager
                 }
                 self.flow_controller.enable(enable);
             }
+            self.flow_controller.update_config(flow_control)?;
```

Comment:
Should we call `flow_controller.update_config(flow_control)` before L79 (`flow_controller.enable(enable)`) ? As it is, I feel like there could be a small race condition and metric issue in the following scenario:

When we disable the flow controller, `checker.reset_statistics()` will be called first, but `current_cfg.enable` may still be `true` when `on_flow_info_msg` is called. So metrics like `SCHED_PENDING_COMPACTION_BYTES_GAUGE` may be non-zero after we disable the flow controller, which might be unexpected. Not sure if I understand all this correctly though.

Reply:
Done.